addons/l10n_ch/models/res_bank.py

```diff
@@ -4,24 +4,38 @@
 import re
 
 from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
 from odoo.tools.misc import mod10r
 
 import werkzeug.urls
 
+ISR_SUBSCRIPTION_CODE = {'chf': '01', 'eur': '03'}
```

Comment:
I personally would have chosen to write currency codes in uppercase, to match with the data in Odoo. That's isn't super grave though